### DISCUSSION TITLE
Livestream over Android

### DISCUSSION THREAD

**Discussion by chicano-music** [datarhei/restreamer#757] [Q&A]
Hey Guys,

Is it possible to stream live from an Android cell phone with its camera via restreamer? If so, how? I've looked everywhere for an answer but haven't found one yet....-.-

I would be very happy about information on this topic! ❤️

Best regards

PS: Thanks for your amazing Work 👍🏽👍🏽👍🏽

**Comment by ioppermann** [comment]
You might checkout [Larix Broadcaster](https://softvelum.com/larix/)

**Reply by chicano-music** [reply]
Thank you for your answer. Of course I know Larix Broadcaster. But I was wondering if I could access restreamer (which runs on my Mac) from my phone and use my phone camera as a connected camera without having to use another service like Larix or streamlabs....

Best regards

**Reply by FilipStadler** [reply]
I use mevo multicam app on phone  it does rtmp to a restreamer where a (OBS in docker  kasem) have scenes I can change and put to another resteamer for outgoing destination from my OBS with overlays etc.   I avoid larix as its not really free, also I did find a app called IRL pro very simular to larix, OBS can also use vdo.ninja scene input so.  I just use this setup now but I have not looked if new sub zero and wbrtc input option to restreamer is possible - the mevo multicam does not run srt but it would improve latency  - anyway you can make a simular setup with OBS on the Mac just to share some ideas :)
